File: tests/language_completion/fixtures/php/control_flow.php
<?php
function sum_to($limit) {
    $index = 0;
    $total = 0;
    while ($index < $limit) {
        $total += $index;
        $index += 1;
    }
    if ($total > 5) {
        return $total;
    }
    return 0;
}

function main() {
    echo "php_flow";
    echo sum_to(5);
}
?>

File: tests/language_completion/fixtures/java/control_flow.java
public class ControlFlow {
    public static int sumTo(int limit) {
        int index = 0;
        int total = 0;
        while (index < limit) {
            total += index;
            index += 1;
        }
        if (total > 5) {
            return total;
        }
        return 0;
    }

    public static void main(String[] args) {
        System.out.println("java_flow");
        System.out.println(sumTo(5));
    }
}
